docs/_scripts/gen_community.py

```diff
@@ -0,0 +1,167 @@
+import os
+import pathlib
+import warnings
+
+import yaml
+from md_utils import dict_to_table
+
+import minari
+from minari.utils import get_dataset_spec_dict
+
+
+DATASET_FOLDER = pathlib.Path(__file__).parent.parent.joinpath("datasets")
+
+
+def _rst_escape(text: str) -> str:
+    """Escape text for RST."""
+    return text.replace("*", r"\*").replace("_", r"\_").replace("`", r"\`")
+
+
+def _generate_community_dataset_page(dataset_entry):
+    """Generate a dataset page for a community dataset."""
+    dataset_id = dataset_entry.get("dataset_id", "")
+    display_name = dataset_entry.get("display_name", dataset_id)
+
+    if not dataset_id:
+        warnings.warn(f"Skipping dataset entry without dataset_id: {dataset_entry}")
+        return
+
+    try:
+        # Get metadata from remote datasets first
+        remote_datasets = minari.list_remote_datasets(latest_version=True)
+        if dataset_id not in remote_datasets:
+            warnings.warn(f"Dataset {dataset_id} not found in remote datasets")
+            return
```

Comment:
nope, this should actually work with datasets that are not in remote. We can download such datasets by using minari.dowload_dataset and using the full path, e.g. `hf://user/dataset